### DISCUSSION TITLE
Successful Outlook Online connection?

### DISCUSSION THREAD

**Discussion by phw198** [phw198/OutlookGoogleCalendarSync#1890] [Polls]
Have you tried to have OGCS sync with an "Outlook Online" mailbox (available from v3.0)?

If you are [being blocked due to admin approval](https://github.com/phw198/OutlookGoogleCalendarSync/discussions/1889), have a look at the [ICS poll](https://github.com/phw198/OutlookGoogleCalendarSync/discussions/1893).

For any other problem, please start a new discussion to help get to the bottom of the issue.

**Comment by phw198** [comment]
For those that were blocked before 16-Nov-2025 due to admin approval, Microsoft have finally verified me as a publisher. I'd be interested if this makes a difference to getting admin approval now, if anyone is able to re-test?

<img width="458" height="777" alt="Image" src="https://github.com/user-attachments/assets/5041127b-7bb3-484e-8fff-98d083330f6d" />

**Reply by AndersMogensenMJBR** [reply]
The access is still blocked, because of our IT department policy. They wont allow third party access :-(
I will stick to v2 :-)

**Reply by phw198** [reply]
Yeah there are multiple levels of access control - one of them is to allow apps from verified publishers, so hopefully it will help some people though many organisations are much stricter than that.

FWIW, you can still run v3 (which is what I'd recommend), just you can't use the Outlook Online functionality that's been added.

**Comment by double-dev23** [comment]
Resurrecting an old thread here, but I can connect to a few of my outlook online accounts no problem!  The issue is when I run multiple instances of OGCS as described in the wiki, it tries to use the same outlook account once authenticated for every instance.  I have several outlook calendars I'm trying to sync together into a google calendar... is there a way to do that using online?  Or do I have to fall back to local?

**Reply by phw198** [reply]
I think this will need some investigation on my part - I've raised #2370 to cover this. Thanks for reporting it 👍🏻
